components/CatCard/CatCard.js

```diff
@@ -0,0 +1,7 @@
+export default function CatCard({ cat }) {
+  return (
+    <>
```

Comment:
you don't need the fragments for a single h3 but i think you'll fill the CatCard maybe in the next user story but I just wanted to say that you don't need these yet

Reply:
changes are committed